bindings/wasm/examples/editor.d.ts

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 declare function show(manifold: Manifold): Manifold;
+declare function only(manifold: Manifold): Manifold;
```

Comment:
Should we put the documentation for `show` and `only` here as well?

Reply:
Good call, done.